packages/react-native/src/primitives/Radio/Radio.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useCallback, useMemo } from 'react';
+import { Pressable, View, ViewStyle } from 'react-native';
+
+import { Label } from '../index';
+import { getFlexDirectionFromLabelPosition } from '../Label/utils';
+
+import { styles } from './styles';
+import { RadioProps } from './types';
+import { getRadioButtonStyles } from './getRadioButtonStyles';
+
+export default function Radio<T>({
+  accessibilityRole = 'radio',
+  disabled,
+  label,
+  labelPosition = 'end',
+  labelStyle,
+  onChange,
+  radioButtonContainerStyle,
+  radioButtonStyle,
+  selected,
+  size = 'medium',
+  style,
+  value,
+  ...rest
+}: RadioProps<T>): JSX.Element {
+  const handleOnChange = useCallback(() => {
+    if (!disabled) {
+      onChange?.(value);
+    }
+  }, [onChange, value, disabled]);
+
+  const containerStyle: ViewStyle = useMemo(
+    () => ({
+      ...styles.container,
+      flexDirection: getFlexDirectionFromLabelPosition(labelPosition),
+      ...(disabled && styles.disabled),
+    }),
+    [disabled, labelPosition]
+  );
+
+  return (
+    <Pressable
+      {...rest}
+      accessibilityRole={accessibilityRole}
+      // hitSlop will be platform-specific,
+      // and partially depends on how much spacing the RadioGroupField will apply
+      hitSlop={5}
```

Comment:
Do we always want this to be `5`? What are the expected touch target sizes for the platforms?

Reply:
Looks like the minimum touch target size for [iOS](https://www.w3.org/WAI/GL/mobile-a11y-tf/wiki/M2#:~:text=iOS%20Human%20Interface%20Guidelines%3A%20Adaptivity,pixels%20wide%2044%20pixels%20tall.) is 44px wide and 44px tall. For [Android](https://www.w3.org/WAI/GL/mobile-a11y-tf/wiki/M2#:~:text=Android%20Developers%3A%20Metrics%20and%20Grids%20recommends%20targets%20be%20at%20least%2048dp%20high%20and%20wide%20so%20that%20they%20will%20never%20be%20smaller%20than%20a%20minimum%20recommended%20target%20size%20of%207mm.) it's 48dp X 48dp. 

In addition to taking the platform into consideration, do we want to dynamically calculate `hitSlop` based on the size of the element? For example, if the Radio is 12 high and 40 wide, then we would want the hitSlop to be something like:

```
hitSlop={{ top: 16, right: 2, bottom: 16, left: 2 }}
```
in order to increase the target size to the minimum. 

However, if the element is larger (e.g., 250 X 250), then adding `hitSlop` wouldn't be necessary. 

Thus I'm wondering, if `hitSlop` is something that we want to handle for all of our touchable/pressable primitives, perhaps we should consider writing a util for it? The util could take both the platform and the size of the element, and return the calculated `hitSlop`.

Thoughts?